### PR TITLE
add osquery tool to incident_response.json

### PR DESCRIPTION
**Pull request by nadasuhailAyesh12**

### CODE REVIEW

File: data/tools/incident_response.json [hunk 1]
```diff
@@ -62,6 +62,16 @@
...
+    {
+      "name": "osquery",
+      "website": "https://osquery.io/",
+      "source": "https://github.com/osquery/osquery",
+      "description": "Endpoint visibility tool providing SQL-based querying for monitoring and security analysis of operating systems.",
+      "language": "C++",
+      "price": "Free",
+      "online": "True"
+    }
+    ,
```
Comment (noraj) [inline data/tools/incident_response.json:74]:
```suggestion
```

File: data/tools/incident_response.json [hunk 2]
```diff
@@ -62,6 +62,16 @@
...
     },
+    {
+      "name": "osquery",
+      "website": "https://osquery.io/",
+      "source": "https://github.com/osquery/osquery",
+      "description": "Endpoint visibility tool providing SQL-based querying for monitoring and security analysis of operating systems.",
+      "language": "C++",
+      "price": "Free",
+      "online": "True"
+    }
```
Comment (noraj) [inline data/tools/incident_response.json:73]:
```suggestion
    },
```

File: data/tools/incident_response.json [hunk 3]
```diff
@@ -62,6 +62,16 @@
       "keywords": "HybridAnalysis, Any.Run, URLhaus, MISP, CAPE, Malshare, Valhalla, Hashlookup, malware",
       "blackarch": "munin-hashchecker"
     },
+    {
+      "name": "osquery",
+      "website": "https://osquery.io/",
+      "source": "https://github.com/osquery/osquery",
+      "description": "Endpoint visibility tool providing SQL-based querying for monitoring and security analysis of operating systems.",
```
Comment (noraj) [inline data/tools/incident_response.json:69]:
```suggestion
      "description": "Endpoint visibility tool providing SQL-based querying for monitoring and security analysis of operating systems",
```

File: data/tools/incident_response.json [hunk 4]
```diff
@@ -62,6 +62,16 @@
...
+      "language": "C++",
```
Comment (noraj) [inline data/tools/incident_response.json:70]:
```suggestion
      "language": "Cplusplus",
```

> The language field value should match [Font Mfizz](http://fizzed.com/oss/font-mfizz) Programming languages name when it exists and be capitalized
>
> Example: Cplusplus for C++ or Nodejs for Node.js

https://inventory.raw.pm/docs/docs/contribution_guidelines/format.html

**Review by noraj — CHANGES_REQUESTED** [review]
Thanks for the contrib

**Comment by nadasuhailAyesh12** [comment]
I have updated the changes.Can you please take a look @noraj

**Comment by noraj** [comment]
@nadasuhailAyesh12 Thanks for your contribution, I'll happily accept new pull request in the future.

**Comment by noraj** [comment]
I squash-merged it here: https://github.com/noraj/rawsec-cybersecurity-inventory/commit/8538d01bc83907e173c539066faeb7ba99edf1a4

**Comment by noraj** [comment]
https://twitter.com/rawsec_cyber/status/1771925413951918375

**Pull request closed without merging**